packages/flutter/lib/src/rendering/sliver_group.dart

```diff
@@ -213,7 +214,11 @@ class RenderSliverMainAxisGroup extends RenderSliver with ContainerRenderObjectM
         double childScrollOffset = 0.0;
         RenderSliver? current = childBefore(child as RenderSliver);
         while (current != null) {
-          childScrollOffset += current.geometry!.scrollExtent;
+          // If the sliver is not clipped, then we add the scroll extent of the
+          // sliver to the childScrollOffset.
+          if (current.geometry?.hasVisualOverflow == false) {
```

Comment:
is this change also applicable for `GrowthDirection.reverse`?

Reply:
Yes yes